src/main/java/org/assertj/core/internal/Objects.java

```diff
@@ -382,6 +383,21 @@ public void assertNotNull(AssertionInfo info, Object actual) {
     throw failures.failure(info, shouldNotBeNull());
   }
 
+  /**
+   * Asserts that the given object is not {@code null}.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the given object.
+   * @param label the label to represent actual in the error message
+   * @throws AssertionError if the given object is {@code null}.
+   */
+  public void assertNotNull(AssertionInfo info, Object actual, String label) {
+    if (actual != null) {
```

Comment:
invert condition and throw the failure: `if (actual != null) throw failures.failure(info, ShouldNotBeNull.of(label));` to make the code lighter

Reply:
`actual == null` I suppose.  Absolute agreed on the idea. But it will not be consistent with the idiom used in all other pair methods of this class (assertNull, assertSame, assertNotSame and so for...). So I will do the modifications for others too.

Reply:
yes :laughing: 
don't worry about consistency for that, ideally I would refactor everything but that's too much work so I focus on the new code or the code being changed.